lib/generators/spotlight/templates/config/initializers/sir_trevor_rails.rb

```diff
@@ -1,10 +0,0 @@
-# frozen_string_literal: true
-
-# Required for 0.6 and up:
-# https://github.com/madebymany/sir-trevor-rails#upgrade-guide-to-v060
-class SirTrevorRails::Block
```

Comment:
this initializer has to be manually removed on existing apps; otherwise you get this error during `db:migrate` https://gist.github.com/dunn/d20dedc996fd7fee7c9261ae72a513dc

are there upgrade notes mentioning this?